etc/config/c++.amazon.properties

```diff
@@ -826,6 +826,12 @@ libs.ctre.versions.trunk.version=trunk
 libs.ctre.versions.trunk.path=/opt/compiler-explorer/libs/ctre/master/include
 libs.ctre.versions.v2.version=v2
 libs.ctre.versions.v2.path=/opt/compiler-explorer/libs/ctre/v2/include
+libs.spy.name=SPY
+libs.spy.description=C++17 constexpr settings detectors
+libs.spy.versions=v003
```

Comment:
Can you add a `trunk` version here too please? (You're synching it so unless you specifically don't want it then)